docker/docker-compose_without-host.yml

```diff
@@ -0,0 +1,23 @@
+# This docker-compose file loads 
+#   nats with JetStream enabled
+#   a local OCI registry
+#   redis (for the kvredis capability provider)
+
+version: "3"
+services:
+  nats:
+    image: nats:2.3
+    ports:
+      - "4222:4222"
+      - "6222:6222"
+      - "8222:8222"
+    command: [ "-js" ]
```

Comment:
It drives me nuts that we can do this in docker compose but not in github actions.